packages/lambdas/src/ihe-gateway-v2-scheduled-report.ts

```diff
@@ -0,0 +1,42 @@
+import { getEnvVarOrFail } from "@metriport/core/util/env-var";
+import { getSecretValueOrFail } from "@metriport/core/external/aws/secret-manager";
+import { initReadonlyDbPool } from "@metriport/core/util/pg";
+import {
+  generatePatientDiscoveryReport,
+  generateDocumentQueryReport,
+  generateDocumentRetrievalReport,
+} from "@metriport/core/external/carequality/ihe-gateway-v2/report/report";
+import { capture } from "./shared/capture";
+import * as Sentry from "@sentry/serverless";
+
+// Keep this as early on the file as possible
+capture.init();
+
+const dbCredsArn = getEnvVarOrFail("DB_CREDS");
+const dbReadOnlyEndpoint = getEnvVarOrFail("DB_READ_REPLICA_ENDPOINT");
+const region = getEnvVarOrFail("AWS_REGION");
+
+capture.setExtra({ lambdaName: "scheduled-report-lambda" });
+
+export const handler = Sentry.AWSLambda.wrapHandler(async () => {
```

Comment:
Either use Sentry's default handler or report to Sentry on a "catch all" error handling code, but not both - otherwise we get two notifications for each error.

I suggest the latter, since it's more detailed.

Example: https://github.com/metriport/metriport/pull/2352/files#diff-6380b83d42c093e8b76d7b5c68503daf640b7a59eb672c51dedd478b6ce230ebR134